### DISCUSSION TITLE
Integrate with cloudinary as additional media storage

### DISCUSSION THREAD

**Discussion by hengage** [withlogicco/django-prose#97] [Q&A] [answered]
I'd like to propose an idea for integrating Cloudinary as an additional media storage option alongside the default media backend. This would allow users to upload images to Cloudinary, receive the uploaded image URL, and render it as an image in the editor.

The idea is to make this integration configurable, so users can choose to use Cloudinary as an additional storage option or stick with the default media backend.

**Comment by akalipetis** [comment]
Starting here, I ended up in this article: https://cloudinary.com/blog/managing-media-files-in-django

Wouldn't it make more sense to have Cloudinary media as a standalone thing to upload images and then use those in Django prose instead? How would you use Cloudinary differently in your proposal?

**Reply by hengage** [reply]
Yeah this is exactly what i meant. Sorry for the late reply, a lot on my plate.

**Reply by akalipetis** [reply, marked as answer]
No worries, I believe what would be the best idea here is to allow Django Prose to use a different storage backend than the default. Then, if you want to use Cloudinary for example, you can create a storage backend like the one described in the article and use that one.

**Reply by hengage** [reply]
That's it!

**Reply by parisk** [reply]
Issue created here: https://github.com/withlogicco/django-prose/issues/98 😄
